examples/azure/terraform-azure-cosmosdb-example/README.md

```diff
@@ -0,0 +1,30 @@
+# Terraform Azure CosmosDB Example
+
+This folder contains a Terraform module that deploys resources in [Azure](https://azure.microsoft.com/) to demonstrate how you can use Terratest to write automated tests for your Azure Terraform code.
+
+This module deploys a [CosmosDB](https://azure.microsoft.com/services/cosmos-db/) instance.
```

Comment:
Can you follow the description format we used in VM example here https://github.com/gruntwork-io/terratest/blob/master/examples/azure/terraform-azure-vm-example/README.md

Reply:
I have updated the README to clarify the resources configured by the Terraform template.